elytron/WFCORE-5279-elytron-ssl-subsystem.adoc

```diff
@@ -0,0 +1,196 @@
+= [WFCORE-5279] Split Elytron SSL into its own subsystem
+:author-mmazanek:   Martin Mazanek
+:email-mmazanek:    mmazanek@redhat.com
+:author:            Cameron Rodriguez
+:email:             carodrig@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Working on bootable jar layers, it is apparent that TLS requirements for a server can be distinct from the general
+authentication requirements, and also that the Elytron subsystem takes a number of resources to define a working TLS
+configuration.
+
+This RFE is to look at providing a new subsystem dedicated to a simplified TLS configuration. The Elytron subsystem
```

Comment:
Do we plan to replace the default TLS configuration with this new one?

Reply:
Not at the moment, this is just planned to be a simpler way of setting up a TLS configuration. Elytron's existing code would remain for integrating with security domains, mappers, etc. when needed